### ISSUE TITLE
TypeError: a bytes-like object is required, not 'str'

### ISSUE THREAD

**Issue by anthonyho** [plotly/dash-auth#2]
Hi there, 

I ran into some error while trying to reproduce the "Basic Auth Example" as detailed [here](https://plot.ly/dash/authentication). I installed dash, dash-auth and plotly, and copied and pasted the auth example into a script called `test.py`. The error that I ran into is as follows:

```
> python test.py
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 159-397-137
 * Running on http://127.0.0.1:8050/ (Press CTRL+C to quit)
127.0.0.1 - - [18/Aug/2017 10:29:49] "GET / HTTP/1.1" 401 -
127.0.0.1 - - [18/Aug/2017 10:29:55] "GET / HTTP/1.1" 500 -
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\ProgramData\Anaconda3\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\ProgramData\Anaconda3\lib\site-packages\dash_auth\auth.py", line 19, in wrap_index
    if self.is_authorized():
  File "C:\ProgramData\Anaconda3\lib\site-packages\dash_auth\basic_auth.py", line 16, in is_authorized
    header.split('Basic ')[1]).split(':')
TypeError: a bytes-like object is required, not 'str'
127.0.0.1 - - [18/Aug/2017 10:29:55] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
127.0.0.1 - - [18/Aug/2017 10:29:55] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
127.0.0.1 - - [18/Aug/2017 10:29:55] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
127.0.0.1 - - [18/Aug/2017 10:29:56] "GET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
127.0.0.1 - - [18/Aug/2017 10:29:56] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
```

For reference, I'm running this with Python 3 on Windows 7 Enterprise, with the following versions of dash and plotly:
```
dash.__version__
Out[7]: '0.17.7'

plotly.__version__
Out[8]: '2.0.12'
```

I'd love some help in figuring out what's going on!

Thanks,
Anthony

**Comment by chriddyp** [comment]
fixed in https://github.com/plotly/dash-auth/pull/3

**Comment by chriddyp** [comment]
Thanks for reporting @anthonyho! This has been fixed in #3. Upgrade with:
```
pip install dash-auth==0.0.5
```

**Comment by anthonyho** [comment]
Works like a charm! Thank you!
